source/agora/test/Transaction.d

```diff
@@ -0,0 +1,167 @@
+/*******************************************************************************
+
+    Contains tests for transaction verification
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.test.TransactionManager;
```

Comment:
Those unittests don't use the network. They are good though, and should be moved to the `TransactionManager` code. They are "simple" `unittest` as opposed to ["Network unittests"](https://github.com/bpfkorea/agora/pull/74/files#diff-f69cd4b5588d09bb541f85596fdacad3R193).

Reply:
Modified